test/unit/api/opterm_black.h

```diff
@@ -55,3 +60,117 @@ void OpTermBlack::testIsNull()
+  uint32_t floatingpoint_to_sbv_total_idx = floatingpoint_to_sbv_total_ot.getIndices<uint32_t>();
+  TS_ASSERT(floatingpoint_to_sbv_total_idx == 14);
+
+  OpTerm tuple_update_ot = d_solver.mkOpTerm(TUPLE_UPDATE_OP, 5);
+  uint32_t tuple_update_idx = tuple_update_ot.getIndices<uint32_t>();
+  TS_ASSERT(tuple_update_idx == 5);
+}
+
+void OpTermBlack::testPairUintIndices()
+{
+    OpTerm bitvector_extract_ot = d_solver.mkOpTerm(BITVECTOR_EXTRACT_OP, 4, 0);
+  std::pair<uint32_t, uint32_t> bitvector_extract_indices = bitvector_extract_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((bitvector_extract_indices == std::pair<uint32_t, uint32_t>{4, 0}));
+
+  OpTerm floatingpoint_to_fp_ieee_bitvector_ot = d_solver.mkOpTerm(FLOATINGPOINT_TO_FP_IEEE_BITVECTOR_OP, 4, 25);
+  std::pair<uint32_t, uint32_t> floatingpoint_to_fp_ieee_bitvector_indices = floatingpoint_to_fp_ieee_bitvector_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((floatingpoint_to_fp_ieee_bitvector_indices == std::pair<uint32_t, uint32_t>{4, 25}));
+
+  OpTerm floatingpoint_to_fp_floatingpoint_ot = d_solver.mkOpTerm(FLOATINGPOINT_TO_FP_FLOATINGPOINT_OP, 4, 25);
+  std::pair<uint32_t, uint32_t> floatingpoint_to_fp_floatingpoint_indices = floatingpoint_to_fp_floatingpoint_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((floatingpoint_to_fp_floatingpoint_indices == std::pair<uint32_t, uint32_t>{4, 25}));
+
+  OpTerm floatingpoint_to_fp_real_ot = d_solver.mkOpTerm(FLOATINGPOINT_TO_FP_REAL_OP, 4, 25);
+  std::pair<uint32_t, uint32_t> floatingpoint_to_fp_real_indices = floatingpoint_to_fp_real_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((floatingpoint_to_fp_real_indices == std::pair<uint32_t, uint32_t>{4, 25}));
+
+  OpTerm floatingpoint_to_fp_signed_bitvector_ot = d_solver.mkOpTerm(FLOATINGPOINT_TO_FP_SIGNED_BITVECTOR_OP, 4, 25);
+  std::pair<uint32_t, uint32_t> floatingpoint_to_fp_signed_bitvector_indices = floatingpoint_to_fp_signed_bitvector_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((floatingpoint_to_fp_signed_bitvector_indices == std::pair<uint32_t, uint32_t>{4, 25}));
+
+  OpTerm floatingpoint_to_fp_unsigned_bitvector_ot = d_solver.mkOpTerm(FLOATINGPOINT_TO_FP_UNSIGNED_BITVECTOR_OP, 4, 25);
+  std::pair<uint32_t, uint32_t> floatingpoint_to_fp_unsigned_bitvector_indices = floatingpoint_to_fp_unsigned_bitvector_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((floatingpoint_to_fp_unsigned_bitvector_indices == std::pair<uint32_t, uint32_t>{4, 25}));
+
+  OpTerm floatingpoint_to_fp_generic_ot = d_solver.mkOpTerm(FLOATINGPOINT_TO_FP_GENERIC_OP, 4, 25);
+  std::pair<uint32_t, uint32_t> floatingpoint_to_fp_generic_indices = floatingpoint_to_fp_generic_ot.getIndices<std::pair<uint32_t, uint32_t>>();
+  TS_ASSERT((floatingpoint_to_fp_generic_indices == std::pair<uint32_t, uint32_t>{4, 25}));
+}
+
+void OpTermBlack::testGetIndicesExceptions()
```

Comment:
I'd prefer having these checks in the corresponding functions above rather than in a separate function (that's the pattern used in all the API unit tests so far).

Reply:
Fixed with 02da5ec1ab49dc09625bd26e005489bd4fba0ed3